### DISCUSSION TITLE
十一月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1568] [通知]
## 更改记录

这次更新修复了依赖注入的一些根源性问题，可能对有闪退情况的同学有所帮助

## 🐛 Bug 修复

- 修复直播退出全屏的快捷键没有生效的问题 @Richasy (#1567)
- 修复因用户Id过长导致的解析问题 @Richasy (#1566)

## 🧰 管理与维护

- 修复因DI使用不当造成的多实例创建的问题 @Richasy (#1565)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2211.1.0'>十一月更新</a>.</em>

**Comment by ll402076216** [comment]
个人动态视频和综合动态无法加载

**Reply by ll402076216** [reply]
![QQ截图20221117150322](https://user-images.githubusercontent.com/69185708/202379176-bbf8a6a6-d4af-488b-b196-b7457524dd7a.png)

**Reply by ghost** [reply]
修一下啊，云之幻

**Reply by shanruihuan** [reply]
我也是这个问题

**Reply by LittleChest** [reply]
有开本地代理（Clash之类的）吗？关掉试试
我正常使用无问题
可以搜一下“UWP回环限制”来了解如何禁用限制

**Reply by ll402076216** [reply]
> 有开本地代理（Clash之类的）吗？关掉试试 我正常使用无问题 可以搜一下“UWP回环限制”来了解如何禁用限制
把哔哩屏蔽了还是不行，之前也开开关关用着没有影响

**Reply by LittleChest** [reply]
那不知道 可以关闭系统代理看下

**Reply by czn1007** [reply]
同样的问题，无法解决

**Reply by aben126** [reply]
我今天下载安装的软件，但发现不能登录，还有搜索视频也不行，最后就是好多栏目打开都显示没视频

**Comment by muxueBlue** [comment]
<img width="584" alt="屏幕截图 2022-11-18 151338" src="https://user-images.githubusercontent.com/102411475/202886100-09ba0e17-2fba-4de3-90db-94cabb701c33.png">
我电脑好像出问题了。这个怎么办啊？不会弄欸。

**Comment by xifengaaa** [comment]
![image](https://user-images.githubusercontent.com/97461803/203488764-9b5c4643-5cf2-40bd-afc2-b49504f8082a.png)
作者大大，这个是怎么回事啊，安全证书手动也安装不上 @Richasy

**Reply by xifengaaa** [reply]
因为证书日期的问题，自己手动改一下电脑的时间就行了，大佬记得发的时候改一下呗o(TヘTo)

**Comment by DIDUJACK** [comment]
![屏幕截图(29)](https://user-images.githubusercontent.com/92197271/206633260-315ee395-0373-43bf-b13e-33247121e3ce.png)
大佬，这是什么问题啊？

**Reply by xifengaaa** [reply]
你先手动看一下证书的属性，作者把证书授信日期设置成之前的一个时间段了，你把电脑的日期自动校准关了，手动改正为证书的时间之内安装就可以了，安装完成之后再校准时间。

**Comment by hekeledeJudy** [comment]
目前遇到推荐页面内容：推荐视频请求失败，账号菜单无法显示，一直转圈，请大佬更新修复~

**Reply by ztime792420442** [reply]
同样的问题

**Reply by NIUGPT-Chat** [reply]
+1

**Comment by NYMYASGP** [comment]
![企业微信截图_20221219162617](https://user-images.githubusercontent.com/121013182/208380737-88300444-5389-4af4-9860-dd7b247f65eb.png)
有没有大佬知道这个怎么解决？

**Reply by xifengaaa** [reply]
你先手动看一下证书的属性，作者把证书授信日期设置成之前的一个时间段了，你把电脑的日期自动校准关了，手动改正为证书的时间之内安装就可以了，安装完成之后再校准时间。
